Security-Hub/manual-install.md

```diff
@@ -0,0 +1,318 @@
+      | FIG_SSL_VERIFY | String | `True` - Enable / Disable SSL verification boolean |
+      | FIG_API_BASE_URL | String | `https://api.crowdstrike.com` - The base URL for the CrowdStrike Falcon API. |
+
+## Lambda
+
+Open the AWS Management Console, then go to the [Lambda dashboard](https://console.aws.amazon.com/lambda/home).
+
+### Create Lambda Layer
+
+1. **Create Layer**:
+    - Click on **Layers** from the navigation pane, then click the **Create layer** button.
+    - Name your layer (e.g., `falconpy-layer`).
+    - Select **Upload a .zip file** and upload the `falconpy-layer.zip` file.
+    - Under **Compatible runtimes**, select `Python 3.7`.
+    - Click the **Create** button.
+
+### Create Lambda Function
+
+1. **Create Function**:
+    - Click on **Functions** from the navigation pane, then click the **Create function** button.
+    - Select **Author from scratch**.
+    - Name the function something descriptive, like `SecHub-Lambda`.
+    - Select **Python 3.7** as the runtime.
+    - Click the **Change default execution role** dropdown and select **Use an existing role**.
+    - Select the Lambda role you created earlier (ex: `Lambda-SecHubRole`).
+    - Click the **Create function** button.
+1. **Upload Function Code**:
+    - Under the Function code section, click on the **Upload from** dropdown and select **.zip file**.
+    - Select and upload the `sechub-identify-detections_lambda.zip` file.
+    - Click the **Save** button.
+1. **Update Handler**:
+    - In the **Code** tab, scroll down to the **Runtime settings** section.
+    - Click on the **Edit** button.
+    - Change the **Handler** to `main.lambda_handler`.
+1. **Configure General Settings**:
+    - In the **Configuration** tab, click on **General configuration**.
+    - Set the **Memory (MB)** to `128` and **Timeout** to `1 min`
+    - Click the **Save** button.
+1. **Configure Environment Variables**:
+    - In the **Configuration** tab, click on **Environment variables**.
+    - Click on **Edit** and add a new environment variable with the key as `DEBUG` and value as `True`.
+    - Click the **Save** button.
+1. **Add Lambda Layer**:
+    - In the **Code** tab, scroll down to the **Layers** section.
+    - Click on **Add a layer**.
+    - Select **Custom layers** and select the layer you created earlier (ex: `falconpy-layer`) and version.
+    - Click the **Add** button.
+
+### Create Lambda Trigger
+
+1. **Create Trigger**:
+    - In the **Function overview** window or under **Configuration -> Triggers**, select the **Add trigger** button.
+    - Select **SQS** as the trigger type.
+    - Select the main SQS queue you created earlier (ex: `SecHubQueue`).
+    - Set the batch size to `1`.
+    - Click the **Add** button.
+
+## Security Group
+
+> If you already have a security group that you would like to use, you can skip this step. As long as the security group allows SSH from your trusted IP, it should work.
```

Comment:
I see multiple of these in the docs, so wonder if these should be notes or important admonitions?